### ISSUE TITLE
Failed to load resource: net::ERR_CONNECTION_REFUSED

### ISSUE THREAD

**Issue by olcariz** [reuseman/flashcards-obsidian#122]
I have just opened Obsidian this day, and the entire app was frozen and consuming a lot of space with this error on the console : 

![image](https://user-images.githubusercontent.com/36270849/188579842-d946619f-355a-41d6-a64d-4f13e5019798.png)

At this point I am very very annoyed, the app require us to manually sync flashcards to Anki, for what reason does the plugin even try to connect to Anki at startup ? 

This is clearly not a way to design the plugin, plus it is frozing the app, and keep on trying connecting indefinitely. It is also weird because it is the first time I see that behavior at startup.

**Comment by Comprehensive-Jason** [comment]
I agree. Instead of attempting to connect to Anki constantly, the plugin should only try to connect to Anki when we send the <kbd>Sync</kbd> command.

**Comment by reuseman** [comment]
This v1 startup problem is fixed in Flashcards 2.0.0. Loading the plugin does not connect to AnkiConnect or retry a failed connection. Anki is contacted only when you run an Anki-related Flashcards command, so an unavailable Anki instance cannot create a startup connection loop. Release: https://github.com/reuseman/flashcards-obsidian/releases/tag/2.0.0

**Comment by olcariz** [comment]
You are 4 years late ahaha
